http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/RefererConfig.scala

```diff
@@ -0,0 +1,4 @@
+package org.broadinstitute.dsde.workbench.leonardo
+package config
+
+final case class RefererConfig(validHosts: List[String])
```

Comment:
Nit: maybe make this a `Set[String]` since we're checking it with `.contains()`

Reply:
ah yes I also like that better - will update!